roomserver/storage/tables/interface.go

```diff
@@ -235,6 +235,8 @@ func ExtractContentValue(ev *types.HeaderedEvent) string {
 		key = "topic"
 	case "m.room.guest_access":
 		key = "guest_access"
+	case "m.room.server_acl":
+		return string(content)
```

Comment:
This feels wrong, but I understand why you're doing it (all keys in content are required). Needs a comment.